components/elm/src/biogeophys/PhotosynthesisType.F90

```diff
@@ -263,6 +265,12 @@ subroutine InitHistory(this, bounds)
        call hist_addfld1d (fname='PSNSHA', units='umolCO2/m^2/s', &
             avgflag='A', long_name='shaded leaf photosynthesis', &
             ptr_patch=this%psnsha_patch)
+
+       this%vcmax25_top_patch(begp:endp) = spval
+       call hist_addfld1d (fname='VCMAX25TOP', units='umolCO2/m^2/s', &
+            avgflag='A', long_name='vcmax at top canopy at 25oC', &
+            ptr_patch=this%vcmax25_top_patch)
```

Comment:
Let's double check if the default is active or inactive (do we need to specify inactive?)